compose/src/main/kotlin/com/yourssu/handy/compose/Chip.kt

```diff
@@ -0,0 +1,111 @@
+package com.yourssu.handy.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.yourssu.handy.compose.foundation.*
+import com.yourssu.handy.compose.icons.HandyIcons
+import com.yourssu.handy.compose.icons.line.*
+
+enum class ChipSize(
+    internal val chipHeight: Dp
+) {
+    Large(32.dp),
+    Small(24.dp)
+}
+
+enum class ChipState {
+    Unselected,
+    Selected,
+    Disabled
+}
+
+@Composable
+fun Chip(
+    text: String = "",
+    modifier: Modifier = Modifier,
+    chipState: ChipState = ChipState.Unselected,
+    chipSize: ChipSize = ChipSize.Large,
+    leadingIcon: ImageVector? = HandyIcons.Line.InfoCircle,
+    trailingIcon: ImageVector? = HandyIcons.Line.Close,
+    chipOnClick: (() -> Unit)? = null,
+    leadingIconOnClick: (() -> Unit)? = null,
+    trailingIconOnClick: (() -> Unit)? = null
```

Comment:
```suggestion
    chipOnClick: (() -> Unit) = {},
    leadingIconOnClick: (() -> Unit) = {},
    trailingIconOnClick: (() -> Unit) = {}
```

이렇게 해주시면 아래 함수에서도 ?.invoke()없이 사용가능해요
```kotlin
Icon(
                    leadingIcon,
                    tint = textColor,
                    iconSize = IconSize.XS,
                    modifier = Modifier.clickable(
                        enabled = chipState != ChipState.Disabled,
                        //onClick = { leadingIconOnClick?.invoke() }) //이부분을
                        onClick = { leadingIconOnClick() }) //이렇게
                )

```

Reply:
오 처음 알았네요! 감사합니다!

Reply:
타입에 괄호는 없어도 됩니다